### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by wspencermiller**
Adding Ironjawz GHB19 Spell lore for #231

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,6 @@ Feel free to submit a PR for any incorrect/missing rules! I am only human, and t
 + [NiklassMM](https://github.com/NiklasMM)
 + [mmorrison](https://github.com/mmorrison)
 + [wspencermiller](https://github.com/wspencermiller)
-+ [Zirhark](https://github.com/Zirhark)
```
Comment (daviseford) [inline README.md]: Well that's rude :) Make sure you add him back in

File: src/army/ironjawz/spells.ts [hunk 2]
```diff
@@ -1,5 +1,69 @@
...
+        effects: [
+          {
+            name: `Da Great Big Green Hand of Gork`,
+            desc: `Da Great Big Green Hand of Gork has a casting value of 7. If successfully cast, pick 1 friendly IRONJAWZ unit wholly within 24" of the caster, visible to them and more than 3" from any enemy units. Remove that unit from the battlefeld and then set it up on the battlefeld anywhere more than 9" from any enemy units. That unit cannot move in the following movement phase.`,
+            when: [HERO_PHASE],
+          },
+        ],
+    },
+    {
+        name: `Bash ’Em Ladz`,
```
Comment (daviseford) [inline src/army/ironjawz/spells.ts]: Run `node clean.js` to clear up these apostrophes

File: README.md (hunk 1 above)
Comment (wspencermiller) [reply README.md]: Fixed! Must have been a weird issue with rebasing

File: src/army/ironjawz/spells.ts (hunk 2 above)
Comment (wspencermiller) [reply src/army/ironjawz/spells.ts]: Updated

**Pull request merged**